### PR TITLE
Add passes into Current Plan list

### PR DESCRIPTION
**Pull request by pawelacio**
- active passes feature was added (now passes will be visible on current plan page)
- few small style alignments

image

### CODE REVIEW

File: src/components/Offer/OfferStyled.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const StyledOfferBody = styled.div.attrs(() => ({
   `}
 `;
 
-export const SubscriptionCardWrapperStyled = styled.section.attrs(() => ({
+export const OfferCardWrapperStyled = styled.section.attrs(() => ({
   className: 'msd__subscription-card'
```
Comment (iwonakulacz) [inline src/components/Offer/OfferStyled.js:33]: we could change it to `msd__offer-card`

File: src/components/OfferCard/OfferCard.js [hunk 2]
```diff
@@ -32,8 +32,7 @@ const SubscriptionCard = ({
   paymentMethod,
   t
 }) => {
-  const isSubscription =
-    getData('CLEENG_OFFER_TYPE') === 'S' || isSubscriptionOffer;
+  const isSubscription = offerType === 'S' || isSubscriptionOffer;
```
Comment (iwonakulacz) [inline src/components/OfferCard/OfferCard.js:35]: do we need `isSubscriptionOffer`? Maybe we could base only on offerType?

Comment (pawelacio) [reply src/components/OfferCard/OfferCard.js]: `isSubscriptionOffer` was removed from the component

File: src/components/Offer/OfferStyled.js (hunk 1 above)
Comment (pawelacio) [reply src/components/Offer/OfferStyled.js]: Changed

**Review by mirydz — APPROVED** [review]

**Pull request merged**